netbox/netbox/views.py

```diff
@@ -47,7 +43,7 @@
     # Circuits
     ('provider', {
         'queryset': Provider.objects.annotate(
-            count_circuits=Count('circuits')
+            count_circuits=Count('circuits', distinct=True)
```

Comment:
We should probably focus only on virtual chassis for this PR/issue, and open separate issues for similar issues that have been identified.